release-notes/9.0/patch-release-index.json

```
@@ -1,41 +1,43 @@
 {
   "channel-version": "9.0",
+  "latest-release": "9.0.0-preview.6",
+  "latest-release-date": "2024-07-09",
   "releases": [
     {
       "release-version": "9.0.0-preview.6",
       "release-date": "2024-07-09",
       "security": false,
-      "release.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata//9.0/preview/preview6/release.json"
+      "release-info-uri": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata//9.0/preview/preview6/release.json"
```

The URI seems to be a bit strange, maybe an issue in the generator (if you generated it).
```suggestion
      "release-info-uri": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/preview/preview6/release.json"
```

---

Thanks. Will check the generator.

---

Fixed.